release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2879,6 +2879,38 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.18
+[id="ocp-4-17-18_{context}"]
+=== RHSA-2025:1703 - {product-title} {product-version}.18 bug fix, and security update advisory
+
+Issued: 26 February 2025
+
+{product-title} release {product-version}.18 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:1703[RHSA-2025:1703] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:1706[RHBA-2025:1706] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.17.18 --pullspecs
+----
+
+[id="ocp-4-17-18-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, the control plane Operator did not honor the set `*_PROXY` environment variables when it checked the API endpoint availability. With this release, the issue is resolved. (link:https://issues.redhat.com/browse/OCPBUGS-50596[*OCPBUGS-50596*]) 
```

Comment:
I think it's the asterisk here. I experienced a bit of trouble with asterisks in the past. You could consider using the escape character to overcome this issue or just drop the asterisk.

https://docs.asciidoctor.org/asciidoc/latest/subs/prevent/